cmd/keytransparency-server/frontend.go

```diff
@@ -122,6 +121,14 @@ func grpcHandlerFunc(grpcServer *grpc.Server, otherHandler http.Handler) http.Ha
 	})
 }
 
+func connectToLog(endpoint string, opts []grpc.DialOption) (trillian.TrillianLogClient, error) {
```

Comment:
How about using `opts ...DialOption` as an argument? This will allow you to call connetToLog with no dial options instead of creating an empty slice, e.g., line 213 below.